test/src/main/java/io/camunda/zeebe/test/exporter/ExporterIntegrationRule.java

```diff
@@ -228,6 +228,22 @@ public ExporterIntegrationRule configure(final InputStream yaml) {
     return configure(Collections.singletonMap(id, config));
   }
 
+  /**
+   * Configures the broker to use the given exporter.
+   *
+   * @param id the exporter ID
+   * @param exporterClassName the exporter class's fully qualified name
+   * @param arguments the arguments to pass during configuration
+   */
+  public ExporterIntegrationRule configure(
```

Comment:
This was necessary to avoid having an explicit dependency on the `DebugHttpExporter` in the `protocol-jackson` module, which would cause me to have a dependency on the broker module, which is just silly. :clown_face: